libpd_wrapper/util/z_queued.h

```diff
@@ -18,24 +18,71 @@ extern "C"
 {
 #endif
 
+/// set the queued print receiver hook, NULL by default
+/// note: avoid calling this while DSP is running
```

Comment:
"Avoid calling" sounds a bit fuzzy. "Do not call" seems clearer.

Reply:
Done.